client/components/EventScheduleSummary/index.jsx

```diff
@@ -17,47 +20,35 @@ export const EventScheduleSummary = ({schedule}) => {
     const end = get(schedule, 'end');
     const interval = get(schedule, 'recurring_rule.interval');
 
-    let startStr = moment(start).format('MMMM Do YYYY, h:mm:ss a');
-    let endStr = moment(end).format('MMMM Do YYYY, h:mm:ss a');
+    let eventDateText;
+
+    if (start.isSame(end, 'day')) {
+        eventDateText = start.format('DD/MM/YYYY @ HH:mma') + ' - ' +
+            end.format('HH:mm a');
+    } else {
+        eventDateText = start.format('DD/MM/YYYY @ HH:mma') + ' - ' +
+            end.format('DD/MM/YYYY @ HH:mm a');
```

Comment:
It would be better to use `config.view.dateformat` and `config.view.timeformat` to better handle internationalization with date/time strings (also the extra `a` in the format string is not required, as the hour format here is 24 hour)